packages/react-notion-x/src/block.tsx

```diff
@@ -652,6 +652,13 @@ export const Block: React.FC<BlockProps> = (props) => {
 
           <div className='notion-callout-text'>
             <Text value={block.properties?.title} block={block} />
+            <>
+              {
+                block.content?.map((elementId) => (
+                  <Block block={recordMap.block[elementId].value} level={1} />
+                ))
+              }
+            </>
```

Comment:
This can actually be
```
<div className='notion-callout-text'>
  <Text value={block.properties?.title} block={block} />
  {children}
</div>
```
Because the NotionRenderer which is the parent already renders block content as children.
This is how the sync block works as well if you want to take a look at that for example.

Reply:
Oh, that's right. Just updated!